### ISSUE TITLE
Matches history

### ISSUE THREAD

**Issue by alexandrutatarciuc** [ronnigaba/BoyumFoosballStats-2.0#13]
I would like to be able to save matches to Cosmos and also fetch them and present them (like the Match Overview page in the previous app). 
But I am not sure how to access Cosmos to be able to test if what I'm doing is working.

**Comment by alexandrutatarciuc** [comment]
@ronnigaba I think this one should be a priority right now, because it will turn the app into a usable one :)

**Comment by ronnigaba** [comment]
@alexandrutatarciuc That already works, there is a CrudService for both Matches and Players.
All Players and Matches have been migrated from the old system, but once we go live, we will have to do a clean migration of matches again.

**Comment by ronnigaba** [comment]
I can try to give you access to the CosmosDb directly in the Azure Portal as well!

**Comment by alexandrutatarciuc** [comment]
@ronnigaba This is what I have so far
![image](https://github.com/ronnigaba/BoyumFoosballStats-2.0/assets/56069835/794a6be9-7e1d-463f-8bd5-c469ee0e13e7)
![image](https://github.com/ronnigaba/BoyumFoosballStats-2.0/assets/56069835/ed7b1ac3-e46b-4cdb-8c28-7c4643cde75d)
What do you think?

**Comment by ronnigaba** [comment]
I think I like the top one better - it's less compact, but I don't think that's a problem.

**Comment by alexandrutatarciuc** [comment]
![image](https://github.com/ronnigaba/BoyumFoosballStats-2.0/assets/56069835/48bb518e-2191-4851-b407-b7d268242c58)

I tried prototyping with how we could show who is what team. Coloring the icons doesn't look good imo. But having the column header saying explicitly what team the column represents seems like a good option for now

**Comment by ronnigaba** [comment]
I agree, it is too subtle - if the teams had been more colorful, it might've worked. 
Let's stick with the header option for now!
